piyushgarg-dev/piyushgargdev-nextjs
# Zoom in property is implemented

## What does this PR do?

Zoom in property is implemented on the gears page.
Fixes #797 


![Screenshot (166)](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/60051901/c228cd91-fd0b-47f0-83b1-143990aca0de)

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

To Reproduce

->Go to website
->Click on 'My Gears'
->See hover effect on content
